src/combinator.cpp

```diff
@@ -93,6 +201,8 @@ void Combinator::on_transformer_config(const metricq::json& config)
         // Register the combined metric as a new source metric
         auto& metric = (*this)[combined_name];
 
+        metric.metadata["displayExpression"] = displayExpression(combined_expression);
```

Comment:
If displayExpression raises an exception, the more important other stuff will also crash. I'd rather put a try-catch-log around this.